slices/libx11-data.yaml

```diff
@@ -0,0 +1,15 @@
+package: libx11-data
+
+essential:
+  - libx11-data_copyright
+
+slices:
+  data:
+    contents:
+      /usr/share/X11/locale/**:
```

Comment:
Are we sure that this package is the only one that puts file in this directory? If so, this glob is fine. Otherwise, Chisel would report a conflict in file paths.

Reply:
Since I am in doubt, I will just put all locales to the respective folders and include them.

Reply:
@rebornplusplus fixed the linting issues and specified all files explicitly, so we're sure glob won't interfere.